### PR TITLE
add a preserveDrawingBuffer (default false) setting

### PR DESCRIPTION
**Pull request by spencermize**
Allows for export using tool like html2canvas

### CODE REVIEW

File: src/js/points.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ var Points = function Points(settings) {
     canvas.className += ' ' + settings.className;
   }
 
-  this.gl = canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
+  this.gl = canvas.getContext('webgl',{preserveDrawingBuffer: settings.preserveDrawingBuffer}) || canvas.getContext('experimental-webgl');
```
Comment (robertleeplummerjr) [inline src/js/points.js]: Can you add this to both calls to `getContext` for continuity?

**Comment by robertleeplummerjr** [comment]
Looks like just one conflict, if you could resolve that, this is good to go.

**Comment by spencermize** [comment]
Sorry - looks like you updated and I didn't catch it!

**Comment by spencermize** [comment]
Sorry - I had just realized that I borked everything up so I re-committed
just the changes I made and merged the .gitignore from your repo.

**Pull request merged**